### DISCUSSION TITLE
[report] works on fusion 360

### DISCUSSION THREAD

**Discussion by TTTPOB** [ysc3839/FontMod#85] [Show and tell]
fusion 360的中文字体很丑，用这个repo是可以替换的，需要在
`%LOCALAPPDATA%\Autodesk\webdeploy\production` 下面的一串hash的文件夹里放置dll和配置。每次更新之后fusion360会新开一个文件夹放程序，所以需要重新把配置和dll拷过去。

**Comment by scrpr** [comment]
@TTTPOB 请问是否能提供一份实例配置文件以供参考？在我尝试时虽然能正常生成`FontMod.yaml`，但是没有任何字体被替换，还是很丑的SimSun。即便开启了debug模式也查看不到任何有效信息，log文件只有两行：
```
[DllMain] Load GDI+ address = 7ffea4ba0000, lasterror = 2
ont.lfFaceName="Microsoft YaHei UI"
```
